backend/dataall/core/environment/cdk/environment_stack.py

```diff
@@ -431,6 +432,28 @@ def create_group_environment_role(self, group: EnvironmentGroup, id: str):
             permissions=group_permissions,
         ).generate_policies()
 
+        external_managed_policies = []
```

Comment:
Not from this PR but on L408 I think we are missing a `group_roles.append(...)` for imported env group roles

Reply:
Fixed, it only affected subscriptions that's why we did not spot it